src/mbed_tools/project/_internal/templates/gitignore.tmpl

```diff
@@ -0,0 +1,2 @@
+.mbedbuild
+build/
```

Comment:
Should we use `cmake_build` for the folder name so as not to clash with the old build system's BUILD folder on case-insensitive file systems like what Windows often employs?

Reply:
Normally follow : BUILD folder (caps)